core/src/core/api/memgraph.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Dict, List, Optional
```

Comment:
I would leave a blank line between std lib import and third-party import. Otherwise, the order is good.